model/stats/db.go

```diff
@@ -102,7 +102,7 @@ var (
 )
 
 // Convenient type to use for arrays in pipeline definitions.
-type array []interface{}
+type Array []interface{}
```

Comment:
Most of the changes in this file are for visibility or refactored some functions (I've highlighted where the new functions) , there are no changes in functionality.